internal/pkg/skuba/deployments/ssh/kubeadm.go

```diff
@@ -83,8 +83,11 @@ func kubeadmJoin(t *Target, data interface{}) error {
 }
 
 func kubeadmReset(t *Target, data interface{}) error {
-	_, _, err := t.ssh("kubeadm", "reset", "--cri-socket", "/var/run/crio/crio.sock", "--ignore-preflight-errors", "all", "--force")
-	return err
+	if _, _, err := t.ssh("systemctl", "status", "kubelet"); err == nil {
+		_, _, err = t.ssh("kubeadm", "reset", "--cri-socket", "/var/run/crio/crio.sock", "--ignore-preflight-errors", "all", "--force")
```

Comment:
I wanted to avoid logs from reset strerr. My original thought was that `do reset` if a node was `node joined`/`bootsrapped`, which has kubelet enabled so that I can avoid the following message.  
```
I1017 16:07:36.666901   30342 states.go:35] === applying state kubeadm.reset ===
E1017 16:07:37.790685   30342 ssh.go:198] W1017 23:07:37.765479   21115 reset.go:98] [reset] Unable to fetch the kubeadm-config ConfigMap from cluster: failed to get config map: Get https://10.86.1.172:6443/api/v1/namespaces/kube-system/configmaps/kubeadm-config: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "kubernetes")
E1017 16:07:37.790705   30342 ssh.go:198] W1017 23:07:37.765677   21115 removeetcdmember.go:79] [reset] No kubeadm config, using etcd pod spec to get data directory
```
Anyway, I reverted to its original code.